index.html

```diff
@@ -7,5 +7,41 @@
     <title>DevQuiz</title>
     <link rel="stylesheet" href="./style.css" />
   </head>
-  <body></body>
+  <body>
+    <div class="landing-container">
+      <nav class="navigation landing-nav">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="#">
+              <img
+                class="hero-logo quiz-hero-logo"
+                src="./src/logo/DevQuiz.png"
+                alt=""
```

Comment:
Give some **alt** value don't keep blank

Reply:
it is fixed now